### PR TITLE
Make OpenId rollingtokens configurable

### PR DESCRIPTION
**Pull request by jardg**
Implements  #1355. Suggested in #1327

### CODE REVIEW

File: src/OrchardCore.Modules/OrchardCore.OpenId/Services/OpenIdConfiguration.cs [hunk 1]
```diff
@@ -101,6 +101,15 @@ public void Configure(string name, OpenIddictOptions options)
                 options.AccessTokenHandler = new JwtSecurityTokenHandler();
             }
 
+            if (settings.UseRollingTokens)
+            {
+                options.UseRollingTokens = true;
```
Comment (PinpointTownes) [inline src/OrchardCore.Modules/OrchardCore.OpenId/Services/OpenIdConfiguration.cs]: (nit: you can save 6 lines by doing ˋoptions.UseRollingTokens = settings.UseRollingTokens;`)

File: src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdSettings.Edit.cshtml [hunk 2]
```diff
@@ -13,6 +13,16 @@
...
     </fieldset>
 
+    <fieldset class="form-group" asp-validation-class-for="UseRollingTokens">
+        <div class="form-check">
+            <label class="form-check-label">
+                <input asp-for="UseRollingTokens" class="form-check-input" checked="@Model.UseRollingTokens" />
+                @T["Use Rolling Tokens"]
+            </label>
+        </div>
+        <span class="hint">@T["When this option is enabled, a new refresh token is issued for each refresh token request (and the previous one is automatically revoked)"]</span>
```
Comment (PinpointTownes) [inline src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdSettings.Edit.cshtml]: Should we add a message indicating this option should be enabled by people who know what they are doing?

Comment (sebastienros) [reply src/OrchardCore.Modules/OrchardCore.OpenId/Views/OpenIdSettings.Edit.cshtml]: What kind of message?

**Comment by jardg** [comment]
Sorry I just realized I forgot to include the setting on the OpenIdSettingsStep.cs . I uploaded the change to my branch.Could that be updated too or do I have to make another pr?
Sorry for the inconvenient

**Comment by PinpointTownes** [comment]
@jardg I went ahead and directly pushed your missing commit to the dev branch: https://github.com/OrchardCMS/OrchardCore/commit/c8deba744a22e871eaa8b732f0655e1471e6d4db.

**Pull request merged**